openai_assistant/test/test_annotations.py

```diff
@@ -88,3 +88,8 @@ def test_has_annotations(self):
 
         result = OpenAIAdapter(message_with_citation).has_annotations()
         self.assertTrue(result)
+
+    async def test_cache(self):
+        adapter = OpenAIAdapter(message_with_citation)
+        await adapter.main()
+        self.assertIn("file-A8iZLTqbU5cxrNOWfe4gd404", adapter.files_cache)
```

Comment:
This doesn't work, unless you have the same key. Use TEST_FILE_ID in the env. 

Update line 11: `load_dotenv('../.env', override=True)`.